cmd/main.go

```diff
@@ -166,50 +172,88 @@ func (o *Options) LoadConfiguration(b []byte) error {
 	return nil
 }
 
-func (o *Options) LoadConfigurationFile() error {
-	configPaths := []string{
-		filepath.Join("{CONFIG}", "kubectl-ai", "config.yaml"),
-		filepath.Join("{HOME}", ".config", "kubectl-ai", "config.yaml"),
-	}
+var defaultConfigPaths = []string{
+	"{CONFIG}/kubectl-ai/config.yaml",
+	"{HOME}/.config/kubectl-ai/config.yaml",
+}
```

Comment:
filepath.Join syntax helped to keep these changes compatible with different OSs (for example, windows uses \ instead of / as path separator).

Pl. restore the original change.

Reply:
[`23f4c35` (#244)](https://github.com/GoogleCloudPlatform/kubectl-ai/pull/244/commits/23f4c35cbee8c6c4cc733341d247dc902d78ebfc)